src/platform/silabs/SiWx/ble/BLEManagerImpl.cpp

```diff
@@ -23,17 +23,17 @@
  */
 
 /* this file behaves like a config.h, comes first */
-#include <platform/internal/CHIPDeviceLayerInternal.h>
+#include <platform/internal/CHIPDeviceLayerInternal.h> // nogncheck
```

Comment:
That should not require a `nogncheck`. I don't see any conditional include of this. This device header should also always be present fo all platforms